src/DataShare/DataShare/help/New-AzDataShareDataSetMapping.md

```diff
@@ -210,7 +210,7 @@ Accept wildcard characters: False
 ```
 
 ### -StorageAccountResourceId
-Azure Storage Account ResourceId
+Azure Storage Account ResourceId that is the source of the data being shared.
```

Comment:
Hi @navba-MSFT , thanks for updating description for these parameters. Their descriptions in source code need to be updated at the same time. See https://github.com/Azure/azure-powershell/blob/fc2fa77a49e122e7ab22397f9a652d8370711402/src/DataShare/DataShare/DataSetMapping/NewAzDataShareDataSetMapping.cs#L62 and https://github.com/Azure/azure-powershell/blob/fc2fa77a49e122e7ab22397f9a652d8370711402/src/DataShare/DataShare/DataSetMapping/NewAzDataShareDataSetMapping.cs#L107

Reply:
done.